### ISSUE TITLE
Make it possible to select a Custom Segment using only keyboard

### ISSUE THREAD

**Issue by mattab** [matomo-org/matomo#9616]
Goal of this issue is to improve accessibility in Piwik. In particular, it should be possible, with keyboard only (no mouse), to [select a custom segment](https://piwik.org/docs/segmentation/#switching-between-segments) to apply in the report.

Expected: with keyboard, one can open the Segment editor which lists the segments, and navigate through the segments using the arrow key (or TAB?), and select a custom segment to load (on ENTER).

**Comment by mattab** [comment]
Great work @CheweyZ - Thanks for your help getting this resolved!
